README.md

```diff
@@ -9,7 +9,7 @@
 
 [freeCodeCamp.org](https://www.freecodecamp.org) is a friendly community where you can learn to code for free. It is run by a [donor-supported 501(c)(3) nonprofit](https://donate.freecodecamp.org) to help millions of busy adults transition into tech. Our community has already helped more than 10,000 people get their first developer job.
 
-Our full-stack web development curriculum is completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
+Our full-stack web development and Machine learning with Python curriculum is completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
```

Comment:
The M in machine learning shouldn't be capitalized here. And there's no need to say "with Python"

Reply:
Updated the requested changes.